x/ibc/23-commitment/value.go

```diff
@@ -0,0 +1,128 @@
+package commitment
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store/state"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Mapping is key []byte -> value []byte mapping, possibly prefixed.
```

Comment:
Hmm? This doesn't have a key at all. Is this a `CommitmentPrefix` as defined in the spec?